aip/auth/4114.md

```diff
@@ -0,0 +1,93 @@
+---
+id: 4114
+scope: auth
+state: approved
+created: 2020-07-28
+---
+
+# Device Certificate Authentication via Mutual TLS
+
+Mutual TLS (a.k.a mTLS) authentication enables authentication of both client and server endpoints in a TLS handshake, and offers the solution to enable Context Aware Access for GCP enterprise customers. With Context Aware Access, the transport context considers the client certificate’s state in the transport (TLS) to make authorization decisions. The use of client certificates provides a stronger indication of the access originating from a trusted device. This allows customers to restrict access from untrusted devices.
```

Comment:
You may turn on the auto line wrap in your editor so the line length is consistent across the file.

Reply:
Yes, will format all paragraphs to 80 characters